tests/FakeItEasy.Tests/Core/FakeManagerTests.cs

```diff
@@ -413,6 +413,21 @@ public void Should_clear_all_added_rules_when_calling_clear_user_rules()
         }
 
         [Fact]
+        public void Should_clear_all_recorded_calls_when_calling_clear_recorded_calls()
+        {
+            // Arrange
+            var manager = new FakeManager(typeof(int), 0);
+            manager.RecordCall(A.Dummy<ICompletedFakeObjectCall>());
+            manager.RecordCall(A.Dummy<ICompletedFakeObjectCall>());
+
+            // Act
+            manager.ClearRecordedCalls();
+
+            // Assert
+            manager.GetRecordedCalls().Should().BeEmpty();
+        }

```

Comment:
The new tests are great, and I think you've added them here because you found places we were testing `Fake`. Could I ask a favour, though?
We've been moving toward putting as many tests as is practical in the Specs, with even more of a focus on this when we're [talking about new or changed behaviour](https://github.com/FakeItEasy/FakeItEasy/blob/master/CONTRIBUTING.md#tests). 

How would you feel about replacing this test to the Acceptance tests? The `FakeSpecs` class looks to be a good place.
Then `FakeFacadeTests.ClearRecordedCalls_should_call_clear_recorded_calls_on_manager` could be struck.